app/controllers/concerns/http_auth_concern.rb

```diff
@@ -0,0 +1,12 @@
+module HttpAuthConcern
+  extend ActiveSupport::Concern
+
+  included do
+    if ENV["BASIC_AUTH_USERNAME"].present? && ENV["BASIC_AUTH_PASSWORD"].present?
```

Comment:
@slorek Do we need to add the env vars into Terraform etc...?

Reply:
This has already been done by Vito